.github/workflows/release.yaml

```diff
@@ -0,0 +1,53 @@
+name: Release operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      commitSha:
+        description: Commit SHA
+        required: true
+      operatorVersion:
+        description: Operator version
+        required: true
+      authorinoVersion:
+        description: Authorino version
+        required: true
+        default: latest
+
+jobs:
+  build:
+    name: Release operator
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Install gettext-base
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y gettext-base
+    - name: Set up Go 1.19.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.19.x
+      id: go
+    - name: Checkout code at commit SHA
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.inputs.commitSha }}
+    - name: Create release branch
+      run: |
+        git checkout -b release-v${{ github.event.inputs.operatorVersion }}
+    - name: Prepare release
+      run: VERSION=${{ github.event.inputs.operatorVersion }} AUTHORINO_VERSION=${{ github.event.inputs.authorinoVersion }} GH_TOKEN=${{ secrets.GITHUB_TOKEN }} make prepare-release
```

Comment:
We need to make sure the secret won't leak to the logs.

Reply:
Whilst I can't find exact proof, I _think_ that this `GITHUB_TOKEN` secret should be masked in the runner. I can see an example usage here https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-2-calling-the-rest-api